### ISSUE TITLE
Saved Games

### ISSUE THREAD

**Issue by MrKnitster** [cambragol/advent-of-the-trinity#24]
When you asked for saved game files, I realized that I will potentially want to use those files myself in new betas of the game. Are the 4 you mentioned (items, montsters, outmonst, and party) the only ones that need to be copied over?

**Comment by cambragol** [comment]
Those 4 are the 4 you must keep, or not overwrite, to keep your save game. 

The aforementioned folders contains save games with all items, no items, and a dragon save, for when we get on to, or have the need for, specific testing.

**Comment by TheVaxen** [comment]
I've noticed that there are a number of different SAV file locations: like "cloud_saves", and "Dragon save" etc. Are they backups of the SAV files in the main directory?

**Comment by cambragol** [comment]
Only the 4 save files in the main directory are being used. 

cloud_saves sounds like a GOG thing? I am not sure of that. What are you playing the game on? (system, emulator, etc)

Dragon Save, All Items Save, and No items Save folders contain saves I set up for testing, and included for you guys if we needed to do specific testing, quickly. They are not actively used by the game, but if you wanted to use them you just copy the saves from there out to your main folder and play.
